app/bin/countries.rb

```diff
@@ -44,90 +47,62 @@
 contributors = data['contributors']
 companies = Hash.new(0)
 
-def ensure_company(companies, companies_info, key, title, link)
+def ensure_company(companies, companies_info, key, title)
```

Comment:
While we're changing this can we rename it to `ensure_country`? Or is that too much of a refactor?